src/OpenTelemetry.ResourceDetectors.Container/README.md

```diff
@@ -34,6 +34,66 @@ your application is running:
 
 - **ContainerResourceDetector**: container.id.
 
+## Kubernetes
+
```

Comment:
```
To make container ID resolution work, container and pod name should be provided through KUBERNETES_CONTAINER_NAME and KUBERNETES_POD_NAME environment variable respectively and pod should have at least get permission to kubernetes resource pods.
It can be done by utilizing YAML anchoring, downwards API and RBAC (Role-Based Access Control).

If KUBERNETES_POD_NAME is not provided, detector will use HOSTNAME as a fallback, but it may not work in some environments or if hostname was overridden in pod spec.

Below is an example of how to configure sample pod to make container ID resolution working:

Reply:
Thank you for the suggestion. Done.